src/ploneintranet/workspace/browser/tiles/workspaces.py

```diff
@@ -41,7 +41,7 @@ def my_workspaces(context):
         path=ws_path,
     )
     workspaces = []
-    for brain in brains:
+    for brain in brains[:5]:
         css_class = escape_id_to_class(brain.getId)
```

Comment:
This change not only causes a test regression but is actually invalid. It makes it impossible to reach my workspaces if I'm in more than five.